src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color,double side) {
+        super(color);
+        this.side=side;
```

Comment:
There should be spaces around the assignment operator for better readability: `this.side = side;`.